streams/examples/src/test/java/org/apache/kafka/streams/examples/wordcount/WordCountProcessorTest.java

```diff
@@ -36,19 +36,19 @@
 public class WordCountProcessorTest {
     @Test
     public void test() {
-        final MockProcessorContext context = new MockProcessorContext();
+        final MockProcessorContext<String, String> context = new MockProcessorContext<>();
 
         // Create, initialize, and register the state store.
         final KeyValueStore<String, Integer> store =
             Stores.keyValueStoreBuilder(Stores.inMemoryKeyValueStore("Counts"), Serdes.String(), Serdes.Integer())
                 .withLoggingDisabled() // Changelog is not supported by MockProcessorContext.
                 // Caching is disabled by default, but FYI: caching is also not supported by MockProcessorContext.
                 .build();
-        store.init(context, store);
+        store.init(context.getStateStoreContext(), store);
```

Comment:
Here's where we're switching contexts to the StateStoreContext to invoke the new API.